dts/bindings/dma/dma-pl330.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+description: PL330 DMA Controller
+
+compatible: "dma-pl330"
```

Comment:
The compatible string normally starts with a vendor.  Should this be "arm,dma-pl330" ?

As Linux already has a pl330 DeviceTree definition [1], do you want to align this with the Linux version?

[1]: https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/dma/arm-pl330.txt

https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/dma/arm-pl330.txt

Reply:
ack. Yes, we can align with linux version.